--- PageSettings.qml
+++ PageSettings.qml
@@ -170,6 +170,13 @@
             description: qsTr("Wi-Fi")
             property VeQuickItem accessPoint: VeQuickItem { uid: "dbus/com.victronenergy.platform/Services/AccessPoint/Enabled" }
             subpage: accessPoint.value !== undefined ? wifiWithAP : wifiWithoutAP
             Component { id: wifiWithoutAP; PageSettingsWifi {} }
             Component { id: wifiWithAP; PageSettingsWifiWithAccessPoint {} }
         }
+//////// added for AdBlockSettings
+        MbSubMenu
+        {
+            description: qsTr("AdBlock Settings")
+            subpage: Component { PageSettingsAdBlock {} }
+        }

         MbSubMenu {
             description: qsTr("GSM modem")
