--- PageSettings.qml.source.qml
+++ PageSettings.qml.edited.qml
@@ -197,6 +197,13 @@
             description: qsTr("Package manager")
             subpage: Component { PageSettingsPackageManager {} }
         }
+//////// added for AdBlockSettings
+        MbSubMenu
+        {
+            description: qsTr("AdBlock Settings")
+            subpage: Component { PageSettingsAdBlock {} }
+        }
     }
 }
