--- /data/AdBlockSettings/FileSets/PatchSource/PageSettings.qml.orig
+++ /data/AdBlockSettings/FileSets/PatchSource/PageSettings.qml
@@ -123,6 +123,11 @@
 			Component { id: wifiWithoutAP; PageSettingsWifi {} }
 			Component { id: wifiWithAP; PageSettingsWifiWithAccessPoint {} }
 		}
+////////// added for AdBlockSettings
+		MbSubMenu {
+			description: qsTr("AdBlock Settings")
+			subpage: Component { PageSettingsAdBlock {} }
+		}
 
 		MbSubMenu {
 			description: qsTr("GSM modem")
